sqlserver2022k8s/setup/base/assets/entrypoint/shutdown/0100_DatabaseTearOff.ps1

```diff
@@ -22,8 +22,11 @@ try {
     Disable-ScheduledTask -TaskName "MssqlSystem"
     Disable-ScheduledTask -TaskName "MssqlReleaseMemory"
 
-    # Set all user databases in readonly mode
-    Get-DbaDatabase -SqlInstance $sqlInstance -ExcludeSystem | Set-DbaDbState -ReadOnly -Force;
```

Comment:
The problem: if we take the last TXN backup when the dtaabase is in ReadOnly, this backup will be COPY ONLY, affecting the next iteration of backup sets once the pod starts again.

I believe we could take some sort of action right after the restore of the last COPYONLY txn, i am waiting on this: https://github.com/dataplat/dbatools/commit/52c23bb053626e44150b0c698192520c56a2f91a to see if it can be "fixed".